test-network/network.sh

```diff
@@ -422,6 +437,14 @@ while [[ $# -ge 1 ]] ; do
     CC_INIT_FCN="$2"
     shift
     ;;
+  -cport )
+    CC_SERVER_PORT="$2"
+    shift
+    ;;
+  -cdr )
+    CC_DOCKER_RUN="$2"
```

Comment:
`CC_DOCKER_RUN` and `cdr` isn't very distinguishing, even in the old way chaincode runs in a container!
How about simply `-ccaas`? Actually above it looks like you've called it CCAAS_DOCKER_RUN.

Reply:
The options are all quite short here; bash makes it just that bit harder to make CLI options!

However this specific option is ONLY for the chaincode-as-a-service feature - and indicates if the `test-network` scripts should build and run the docker container, as well as the install/approve/commit etc.  

Reason to include this that for the debug approach (to come shortly) you want to control this yourself.

Reply:
I meant that `cdr` aka CC_DOCKER_RUN could be misconstrued to be the old way or the new way, since a docker container is run in both cases. I'd suggest put ccaas/CCAAS in the flag/variable name to make it clear that this option is for chaincode-as-a-service.

Reply:
@denyeart yes,, it's been updated along those lines